release/models/aft/openconfig-aft-common.yang

```diff
@@ -213,6 +221,20 @@ submodule openconfig-aft-common {
           }
         }
 
+        container gre {
```

Comment:
Should we add a description here that `encapsulate-header` should be set to `GRE` for this to apply?

Reply:
Makes sense. Will add that to the description

Reply:
description update. Let me know if anything else needs to be updated as well.